sql/2020/12_Mobile_Web/mobile_traffic_distribution.sql

```diff
@@ -4,11 +4,27 @@ SELECT
   percentile,
   APPROX_QUANTILES(phoneDensity, 1000)[OFFSET(percentile * 10)] AS pct_traffic_from_mobile,
 FROM
-  `chrome-ux-report.materialized.device_summary`,
+  (
```

Comment:
I'd suggest reverting this change. If the results require more explanation, we can do that in the chapter.

Reply:
> > so I changed the query to only look at websites which have entries for both phone and desktop
> 
> I don't think this is methodologically valid. It's not a bug to have mobile-only websites in the dataset; that's a reflection of real-world web usage.

Not saying its a bug, but the question we want to be asking... is for the average site, how much of their traffic is due to phones? Or in more specific terms, wouldn't people think of this question as: "If my site has both mobile and desktop visitors, what % is due to mobile phones".

Perhaps it'd be worth delving into why there are 2million more sites for phones vs desktop. And what kind of pages these even are.

Reply:
IMO there's nothing special about a site that has 99% mobile vs one that has 100% mobile, so we shouldn't be omitting those with 100% as that will skew/bias the results unnecessarily.  Given how prevalent mobile-only sites are, that seems like all the more reason to account for them in the results.

I do agree that it would be an interesting topic of discussion for the chapter to explore why there may be more mobile sites, eg due to personal computing changes around the world, or mobile-only subdomains, etc. But that's not necessarily something we can answer objectively in the dataset.